.circleci/config.yml

```diff
@@ -19,14 +18,19 @@ jobs:
 
       # Keep cache
       - save_cache:
-          key: yarn-{{ checksum "yarn.lock" }}
+          key: yarn-cache-{{ checksum "yarn.lock" }}
           paths:
             - "node_modules"
 
-      # Test
+      # Lint
+      - run:
+          name: Lint
+          command: yarn lint
+
+      # Tests
       - run:
           name: Tests
```

Comment:
if we call "yarn unit", it should be clearer to rename it by "Unit". what do you think?